src/ctim/lib/time.cljc

```diff
@@ -41,21 +51,15 @@
 
 (def now timestamp)
 
-(def default-expire-date (coerce-to-date (time/date-time 2525 1 1)))
+(def default-expire-date #?(:clj (coerce-to-date (time/date-time 2525 1 1))
+                            :cljs (js/Date. 2525 1 1)))
 
 (defn after?
-  "like clj-time.core/after? but uses java.util.Date"
+  "like clj-time.core/after? but uses java.util.Date/ js Date"
   [left right]
   (time/after? (coerce-to-datetime left)
                (coerce-to-datetime right)))
 
-(defn sql-now []
-  (coerce-to-sql-time (now)))
-
-(def to-sql-time coerce-to-sql-time)
-
-(def from-sql-time coerce-to-date)
-
```

Comment:
Yes since we removed sql support there is no need for this anymore.
also, as suggested previously we should split this code into a common toolset lib shared by our projects.